src/main/java/com/khoros/twitter/resources/PostTweetResource.java

```diff
@@ -41,18 +45,21 @@ public Response postTweet(Message message) {
         try {
             if (text.length() > 0 && text.length() <= CHAR_LIMIT) {
                 Status status = twitter.updateStatus(text);
+                LOGGER.info("Successfully posted tweet: " + text);
                 return Response.status(Response.Status.OK)
                         .entity(status)
                         .type(MediaType.APPLICATION_JSON)
                         .build();
             } else {
+                LOGGER.warn("Failed to post tweet. " + CHAR_LIMIT_STR);
                 statusMessage.setStatus(CHAR_LIMIT_STR);
                 return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
                         .entity(statusMessage)
                         .type(MediaType.APPLICATION_JSON)
                         .build();
             }
         } catch (TwitterException e) {
+            LOGGER.error("Failed to post tweet: " + text + "\nTwitter exception: ", e);
```

Comment:
you likely don't need the Twitter exception part of the statement as it should be implied by the exception statement